WalletWasabi/Legal/Assets/LegalDocumentsWw2.txt

```diff
@@ -1,75 +1,279 @@
+
+    The limitation of liability reflects the allocation of risk between the parties.
+    In no event will we be liable for any indirect, special, incidental, punitive or consequential damages in the cases of exclusion under the MIT license referred to above.
+    Notwithstanding the foregoing provision, in no event shall the aggregate liability of the Service Provider, our subsidiaries, partners, affiliates, agents, employees, licensors, service providers, or subcontractors (if any) for any loss or damage that arises as a result of, or in connection with, any of the occurrences described above which exceed the greater of $100 or the service fees that You paid to us for the Service we provide through the Services during the month in which the incident occurred.
+    Some jurisdictions do not allow certain warranty disclaimers or limitations on liability. Only the disclaimers or limitations that are lawful in the applicable jurisdiction will apply to You and our liability will be limited to the maximum extent permitted by law. However, in this case, it is Your responsibility to verify the content of the General Terms and Conditions applicable to You under the applicable law and to use our services accordingly.
+
+9 MISCELLANEOUS PROVISIONS
+==========================
+
+9.1 SEVERABILITY
+    If for any reason a court of competent jurisdiction finds any provision of these Terms to be invalid or unenforceable, that provision will be enforced to the maximum extent permissible and the other provisions of these Terms will remain in full force and effect.
+
+9.2 ARBITRATION
+    You and the Service Provider agree to arbitrate any dispute arising under or in connection with these Terms or Your use of our Services, except for disputes in which either party seeks equitable and other relief for any alleged infringement or unlawful use of copyrights, trademarks, trade names, logos, trade secrets, patents or other intellectual property rights. Arbitration prevents You from suing in court or from having a jury trial. You and the Service Provider agree to notify each other in writing of any dispute within thirty (30) days of when it arises. Notice to the Service Provider shall be sent to legal@zksnacks.com.
+
+    You and the Service Provider further agree:
+     - to attempt informal resolution prior to any demand for arbitration;
+     - that the seat or legal place of any arbitration will be Republic of Seychelles;
+     - that arbitration will be conducted confidentially by a single arbitrator appointed by law of Republic of Seychelles;
+     - the arbitration shall be conducted in accordance with the rules of International Chamber of Commerce; and
+     - that the arbitration in Republic of Seychelles will have exclusive jurisdiction over any appeals of an arbitration award.
+   Other than class procedures and remedies discussed below, the arbitrator has the authority to grant any remedy that would otherwise be available in court.
```

Comment:
```suggestion
    Other than class procedures and remedies discussed below, the arbitrator has the authority to grant any remedy that would otherwise be available in court.
```